src/main/java/EchoBot.java

```diff
@@ -0,0 +1,154 @@
+import java.awt.BorderLayout;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.util.ArrayList;
```

Comment:
Maybe you can move your EchoBot class under the duke package considering the flow.